lib/URI/_punycode.pm

```diff
@@ -147,58 +147,72 @@ sub min {
 1;
 __END__
 
+=encoding utf8
+
 =head1 NAME
 
 URI::_punycode - encodes Unicode string in Punycode
 
 =head1 SYNOPSIS
 
-  use URI::_punycode;
-  $punycode = encode_punycode($unicode);
-  $unicode  = decode_punycode($punycode);
+  use strict;
+  use warnings;
+  use utf8;
 
-=head1 DESCRIPTION
+  use URI::_punycode qw(encode_punycode decode_punycode);
 
-URI::_punycode is a module to encode / decode Unicode strings into
-Punycode, an efficient encoding of Unicode for use with IDNA.
+  # encode a unicode string
+  my $pcode = encode_punycode('http://☃.net'); # http://.net-xc8g
```

Comment:
Would it be nice to use the same variable names in SYNOPSIS and FUNCTIONS?
($pcode, $unicode VS $punycode, $uni_string)